site2/website-next/src/pages/index.js

```diff
@@ -129,7 +130,13 @@ export default function Home() {
   ];
   useEffect((d) => {
     startWaves();
-
+    var winW = window.outerWidth;
```

Comment:
we can't use `window` directly because the buiding is based on `SSR`, you can have a test by `npm run build`, maybe this `https://docusaurus.io/docs/docusaurus-core#useIsBrowser` can help @Paul-TT